Tests/XCUITests/FirstRunTourTests.swift

```diff
@@ -47,7 +47,6 @@ class FirstRunTourTests: BaseTestCase {
     func testShowTourFromSettings() {
         goToNextScreen()
         tapStartBrowsingButton()
-        app.buttons["urlBar-cancel"].tap()
```

Comment:
On my end, the keyboard is already hidden and `urlBar-cancel` is not visible. @isabelrios Please confirm if this change works for Intel.

Reply:
Yes, this test is working for me with that change.